src/devices/Arduino/ArduinoBoard.cs

```diff
@@ -0,0 +1,323 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
And now you've added the 3 lines headers, it will change for a 2 lines one :-D

Reply:
??? Is this the wrong header?

Reply:
The new header is now just 2 lines see #1242 :-)
```csharp
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
```